README.md

```diff
@@ -69,10 +69,25 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Broker. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-broker/en/latest/installation/index.html#using-sources).
 
-For the projet compilation you need to have conan installed. To install conan you need pip 
+First of all, check if you have these packages installed (Note that packages names come from Centos 7 distribution, so if some packages names don't match on your distribution try to find their equivalent names) :
+
+    git, make, cmake3, python3, python3-pip3, lua-devel, gnutls-devel, rrdtool-devel, openssl-devel, ninja-build.
```

Comment:
cmake instead of cmake3 is better if you don't want epel on centos7.
gnutls-devel and openssl-devel already come with conan.
ninja-build is not needed, it is an alternative to make